tests/integration/utils.py

```diff
@@ -463,3 +463,133 @@ def _validate_expected_dep_file_contents(dep_contents_file: Path, output_dir: Pa
         dep_file = output_dir / "deps" / path
         assert dep_file.exists()
         assert dep_file.read_text() == expected_content
+
+
+TEST_EVERYTHING = object()
+TEST_NOTHING = object()
+# Individual files could be added to the set as well.
+paths_to_code = set((Path("cachi2"), Path("tests/integration")))
+supported_pms = set(("rpm", "yarn", "gomod", "npm", "pip"))
```

Comment:
Can we pull this list from the resolver module from `_package_managers` and `_dev_package_managers` instead of having to update the sequence manually at two places?

Reply:
Oh, absolutely! Thank you!